src/main/kotlin/com/project/goms/domain/outing/common/util/impl/OutingConverterImpl.kt

```diff
@@ -2,20 +2,22 @@ package com.project.goms.domain.outing.common.util.impl
 
 import com.project.goms.domain.account.entity.Account
 import com.project.goms.domain.account.presentation.data.response.StudentNumResponse
-import com.project.goms.domain.outing.entity.Outing
 import com.project.goms.domain.outing.common.util.OutingConverter
-import com.project.goms.domain.outing.usecase.dto.OutingAccountDto
+import com.project.goms.domain.outing.entity.Outing
 import com.project.goms.domain.outing.presentation.data.response.OutingAccountResponse
 import com.project.goms.domain.outing.presentation.data.response.OutingCountResponse
+import com.project.goms.domain.outing.usecase.dto.OutingAccountDto
+import com.project.goms.domain.studentCouncil.presentation.data.response.SearchOutingResponse
+import com.project.goms.domain.studentCouncil.usecase.dto.SearchOutingDto
```

Comment:
전 코멘트에서도 말해드렸다 싶이 외출을 검색하는 기능은 학생, 학생회 두 권한 모두 할 수 있어서 이렇게 studentCouncil 패키지에 dto 객체를 만들고 의존하는건 studentCouncil(학생회)만 할 수 있는 기능같아 별로 좋은거 같지는 않습니다.
```suggestion
import com.project.goms.domain.outing.presentation.data.response.SearchOutingResponse
import com.project.goms.domain.outing.usecase.dto.SearchOutingDto
```
추후 이런식으로 outing 패키지에 넣는게 좋아보입니다!

Reply:
수정했습니다.